packages/main/src/plugin/docker/docker-compatibility.ts

```diff
@@ -46,4 +60,112 @@ export class DockerCompatibility {
 
     this.#configurationRegistry.registerConfigurations([dockerCompatibilityConfiguration]);
   }
+
+  protected getTypeFromServerInfo(info: { OperatingSystem?: string }, podmanInfo: unknown): DockerSocketServerInfoType {
```

Comment:
We cannot have a better typing for `podmanInfo` ?

Reply:
we don't care on the structure, we only want to know if there is data in it

Reply:
then make it a boolean an check beforehand ? having unknown feel weird. Even with the requirement on it, I feel like having `podmanInfo: Info | undefined` would be clearer than unknown

Reply:
but it means that I would have to declare a new interface `Info` to declare what is inside the data where I don't need to know what is inside

Reply:
I will change parameter to be `podmanInfo?: unknown` so it's clear it's optional

Reply:
amended